examples/common_vmseries_and_autoscale/example.tfvars

```diff
@@ -0,0 +1,243 @@
+          }
+          "private_blackhole" = {
+            address_prefix = "10.0.0.16/28"
+            next_hop_type  = "None"
+          }
+        }
+      }
+    }
+    subnets = {
+      "management" = {
+        name                            = "mgmt-snet"
+        address_prefixes                = ["10.0.0.0/28"]
+        network_security_group          = "management"
+        route_table                     = "management"
+        enable_storage_service_endpoint = true
+      }
+      "private" = {
+        name             = "private-snet"
+        address_prefixes = ["10.0.0.16/28"]
+        route_table      = "private"
+      }
+      "public" = {
+        name                   = "public-snet"
+        address_prefixes       = ["10.0.0.32/28"]
+        network_security_group = "public"
+        route_table            = "public"
+      }
+      "appgw" = {
+        name             = "appgw-snet"
+        address_prefixes = ["10.0.0.48/28"]
+      }
+    }
+  }
+}
+
+
+# --- LOAD BALANCING PART --- #
+load_balancers = {
+  "public" = {
+    name                        = "public-lb"
+    network_security_group_name = "example-public-nsg"
+    network_security_allow_source_ips = [
+      #  "x.x.x.x", # Put your own public IP address here  <-- TODO to be adjusted by the customer
+      "0.0.0.0/0",
+    ]
+    avzones = ["1", "2", "3"]
+
+    frontend_ips = {
+      "palo-lb-app1-pip" = {
+        create_public_ip = true
+        in_rules = {
+          "balanceHttp" = {
+            protocol = "Tcp"
+            port     = 80
+          }
+        }
+      }
+    }
+  }
+  "private" = {
+    name    = "private-lb"
+    avzones = ["1", "2", "3"]
+
+    frontend_ips = {
+      "ha-ports" = {
+        vnet_key           = "transit"
+        subnet_key         = "private"
+        private_ip_address = "10.0.0.30"
+        in_rules = {
+          HA_PORTS = {
+            port     = 0
+            protocol = "All"
+          }
+        }
+      }
+    }
+  }
+}
+
+appgws = {
+  "public" = {
+    name       = "public-appgw"
+    vnet_key   = "transit"
+    subnet_key = "appgw"
+    zones      = ["1", "2", "3"]
+    capacity   = 2
+    rules = {
+      "minimum" = {
+        priority = 1
+        listener = {
+          port = 80
+        }
+        rewrite_sets = {
+          "xff-strip-port" = {
+            sequence = 100
+            request_headers = {
+              "X-Forwarded-For" = "{var_add_x_forwarded_for_proxy}"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+
+
+# --- VMSERIES PART --- #
+application_insights = {}
+
+vmseries_version = "10.2.3"
+vmseries_vm_size = "Standard_DS3_v2"
+vmss = {
+  "common" = {
+    name              = "common-vmss"
+    vnet_key          = "transit"
+    zones             = ["1", "2", "3"]
+    bootstrap_options = "type=dhcp-client"
+
+    interfaces = [
+      {
+        name       = "management"
+        subnet_key = "management"
+        create_pip = true
```

Comment:
as discussed in some other place, we do not need this

Reply:
Agreed to keep it. Details on the other thread.